### PR TITLE
[zh] Sync security-2023-005 into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Sync security-2023-005 into Chinese

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/news/security/istio-security-2023-005/index.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+publishdate: 2023-12-12
+keywords: [CVE]
+skip_seealso: true
+---
+
+{{< security_bulletin >}}
+
+Istio 安全委员会最近意识到一种潜在情况，即 Istio CNI
+由于其高级别权限而可能被用作已受感染节点的攻击媒介。该载体涉及在受感染的节点上滥用
+`istio-cni-repair-role` `ClusterRole`，将危害范围从本地至节点扩大到集群范围。
```
Comment (windsonsea) [inline content/zh/news/security/istio-security-2023-005/index.md:18]:
```suggestion
`istio-cni-repair-role` `ClusterRole`，将危害范围从本地节点扩大到集群范围。
```

**Review by windsonsea — COMMENTED** [review]
find a typo

File: content/zh/news/security/istio-security-2023-005/index.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+---
+
+{{< security_bulletin >}}
+
+Istio 安全委员会最近意识到一种潜在情况，即 Istio CNI
+由于其高级别权限而可能被用作已受感染节点的攻击媒介。该载体涉及在受感染的节点上滥用
+`istio-cni-repair-role` `ClusterRole`，将危害范围从本地节点扩大到集群范围。
+
+因此，Istio 维护者正在逐步推出对上述 `ClusterRole` 的变更，
+以减少关闭此潜在攻击载体的权限。在补丁版本中，
```
Comment (hanxiaop) [inline content/zh/news/security/istio-security-2023-005/index.md:21]:
```suggestion
以减少权限，关闭此潜在的攻击载体。在补丁版本中，
```

File: content/zh/news/security/istio-security-2023-005/index.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+角色仅限于基于[所选修复模式](/zh/docs/setup/additional-setup/cni/#race-condition--mitigation)的最低要求。
```
Comment (hanxiaop) [inline content/zh/news/security/istio-security-2023-005/index.md:22]:
```suggestion
角色的权限被限制在基于[所选修复模式](/zh/docs/setup/additional-setup/cni/#race-condition--mitigation)的最低必要要求。
```

File: content/zh/news/security/istio-security-2023-005/index.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+releases: [1.18.0 以及之前的所有版本", "1.18.0 到 1.18.5", "1.19.0 到 1.19.4", "1.20.0"]
+publishdate: 2023-12-12
+keywords: [CVE]
+skip_seealso: true
+---
+
+{{< security_bulletin >}}
+
+Istio 安全委员会最近意识到一种潜在情况，即 Istio CNI
+由于其高级别权限而可能被用作已受感染节点的攻击媒介。该载体涉及在受感染的节点上滥用
```
Comment (hanxiaop) [inline content/zh/news/security/istio-security-2023-005/index.md:17]: attack vector 的翻译好像不统一，有媒介有载体

File: content/zh/news/security/istio-security-2023-005/index.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+`istio-cni-repair-role` `ClusterRole`，将危害范围从本地节点扩大到集群范围。
+
+因此，Istio 维护者正在逐步推出对上述 `ClusterRole` 的变更，
+以减少关闭此潜在攻击载体的权限。在补丁版本中，
+角色仅限于基于[所选修复模式](/zh/docs/setup/additional-setup/cni/#race-condition--mitigation)的最低要求。
+在此之前，无论配置如何，都会授权予所有角色，而且被授权的角色过多。
+
+另一个选项可以通过完全消除 Istio CNI 拥有自定义 RBAC 权限的需要来进一步减轻任何潜在的攻击；
```
Comment (hanxiaop) [inline content/zh/news/security/istio-security-2023-005/index.md:25]:
```suggestion
另一个选项可以通过完全消除 Istio CNI 需要自定义 RBAC 权限的需求，来进一步减轻任何潜在的攻击风险；
```

File: content/zh/news/security/istio-security-2023-005/index.md (hunk 4 above)
Comment (wilsonwu) [reply content/zh/news/security/istio-security-2023-005/index.md]: Fixed

File: content/zh/news/security/istio-security-2023-005/index.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+以减少权限，关闭此潜在的攻击载体。在补丁版本中，
+角色的权限被限制在基于[所选修复模式](/zh/docs/setup/additional-setup/cni/#race-condition--mitigation)的最低必要要求。
+在此之前，无论配置如何，都会授权予所有角色，而且被授权的角色过多。
+
+另一个选项可以通过完全消除 Istio CNI 需要自定义 RBAC 权限的需求，来进一步减轻任何潜在的攻击风险；
+由于这种新方法可能存在风险，因此仅在 Istio 1.21+ 上默认启用。 请参阅下文了解可用的配置选项以及所需的角色：
+
+| 配置                             | 角色         | 错误时的行为                                      | 备注              |
+|--------------------------------|------------|---------------------------------------------|-----------------|
+| `values.cni.repair.deletePods` | DELETE Pod | Pod 已被删除，重新调度后它们将具有正确的配置。                   | 1.20 及更早版本中的默认值 |
```
Comment (hanxiaop) [inline content/zh/news/security/istio-security-2023-005/index.md:30]:
```suggestion
| `values.cni.repair.deletePods` | DELETE Pod | Pod 将被删除，重新调度后它们将具有正确的配置。                   | 1.20 及更早版本中的默认值 |
```

File: content/zh/news/security/istio-security-2023-005/index.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+| `values.cni.repair.labelPods`  | UPDATE Pod | Pod 仅带有标签。用户需要采取手动操作来解决。                    |                 |
```
Comment (hanxiaop) [inline content/zh/news/security/istio-security-2023-005/index.md:31]:
```suggestion
| `values.cni.repair.labelPods`  | UPDATE Pod | 只对 Pod 打标签。用户需要采取手动操作来解决问题。                    |                 |
```

**Review by hanxiaop — COMMENTED** [review]
Almost lgtm

**Review by windsonsea — APPROVED** [review]
/lgtm, thanks.

**Pull request merged**